reqs/migrations/0013_auto_20170131_1829.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.5 on 2017-01-31 18:29
+from __future__ import unicode_literals
+
+from django.db import migrations
+from django.template.defaultfilters import slugify
+
+
+attr_to_keyword = dict(
+    aquisition='Acquisition/Contracts',
+    human_capital='Human Capital',
+    cloud='Cloud',
+    data_centers='Data Centers',
+    cybersecurity='Cybersecurity',
+    privacy='Privacy',
+    shared_services='Shared Services',
+    it_project_management='IT Project Management',
+    software='Software',
+    digital_services='Digital Services',
+    mobile='Mobile',
+    hardware='Hardware/Government Furnished Equipment (GFE)',
+    transparency='IT Transparency (Open Data, FOIA, Public Records, etc.)',
+    statistics='Agency Statistics',
+    customer_services='Customer Services',
+    governance='Governance',
+    financial_systems='Financial Systems',
+    budget='Budget',
+    governance_org_structure='Governance - Org Structure',
+    governance_implementation='Governance - Implementation',
+    data_management='Data Management/Standards',
+    definitions='Definitions',
+    reporting='Reporting'
+)
+keyword_to_attr = {keyword:attr for attr, keyword in attr_to_keyword.items()}
+
+
+def forward(apps, schema_editor):
+    Keyword = apps.get_model('reqs', 'Keyword')
+    KeywordConnect = apps.get_model('reqs', 'KeywordConnect')
+    Requirement = apps.get_model('reqs', 'Requirement')
+    keyword_cache, connections = {}, []
+    for req in Requirement.objects.all():
+        req_keywords = []
+        for field_name, tag_name in attr_to_keyword.items():
+            if getattr(req, field_name):
+                req_keywords.append(tag_name)
+        req_keywords.extend(kw.strip()
```

Comment:
I'm against deeply nested (aka "pointy") code, but I think we can satisfy both of us. Giving it a go